replica/replica.go

```diff
@@ -304,6 +306,8 @@ func (r *Replica) Resize(obj interface{}) error {
 			return err
 		}
 	}
+	byteArray := make([]byte, (sizeInBytes-r.info.Size)/4096)
```

Comment:
what if r.info.Size is > sizeInBytes?

Reply:
This is already handled at the controller from where we get this rest call. But its good to add a check here also. I will add it. Thanks @vishnuitta .